### PR TITLE
allow for base64 encoded k8s ca

### PR DESCRIPTION
**Pull request by jbonzo**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -223,6 +225,10 @@ func start_app(config Config) {
 			cluster.Scopes = []string{"openid", "profile", "email", "offline_access", "groups"}
 		}
 
+		if cluster.K8s_Ca_Pem != "" && cluster.K8s_Ca_Pem_Base64_Encoded != "" {
+			log.Fatal("Cannot specify both k8s_ca_pem and k8s_ca_pem_base64_encoded")
+		}
```
Comment (qrpike) [inline main.go:230]: I would say this isn't a fatal error, I would just choose the first one and log a warning that the second one wasn't used.

**Review by qrpike — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (jbonzo) [reply main.go:230]: hm ok. so give preference to `K8s_Ca_Pem`?

Comment (qrpike) [reply main.go]: I would think so, yea. If someone legitimately provides both, it shouldn't be a fatal.

**Review by qrpike — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (jbonzo) [reply main.go]: Yeah I didn’t see a legitimate case to provide both. But the current logic allows for both the raw cert or the file path. So it makes sense to be nicer about it.

**Pull request merged**